web/modules/custom/config_form/config_form.routing.yml

```diff
@@ -0,0 +1,7 @@
+config_form.admin_api_form:
+  path: '/admin/config-form'
+  defaults:
+    _title: 'Fill the config form'
+    _form: 'Drupal\config_form\Form\ConfigForm'
+  requirements:
+    _permission: 'administer'
```

Comment:
does this work?

Reply:
Yes, ma'am this `administer` permission allow admin to access the page.